src/libs/protootshelpers.js

```diff
@@ -17,6 +17,14 @@ export function normaliseAccountName(name) {
 	return name;
 }
 
+export function accountNameFromURL(url) {
+	const splitURL = url.split("/");
+
+	const username = [splitURL.pop(), splitURL.pop()].join("@");
```

Comment:
huh, what does it do? Why do we need two pop statements?

The URLs are formed like this https://queer.group/@account@example.com/110707717973652941, why do we need the ID? Or do I misunderstand your code? ^^

Reply:
while this is probably technically unnecessary, since we can get the account name from the API, the way things are set up currently we need an account name.
Glitch-soc notifications for likes/boosts don't have a title on the link we can grab the account name from, so we have to work off of the href instead, which is always (famous last words) `https://server.tld/@account`. the two pops give us `[@account, server.tld]`.

this function is only called in [addProplate.addToNotification()](https://github.com/ItsVipra/ProToots/blob/glitch-soc-types/src/content_scripts/protoots.js#L324) and pulls the url argument from the AccountNameEL's href.

Reply:
hm. what about instances that are known to ours as `instance.tld` but are served under `fedi.instance.tld`? Would it work there as well?

Reply:
no, but worst case we have two entries in our cache. i feel like that's an acceptable risk :D